core/common/src/main/java/alluxio/util/executor/ControllableQueue.java

```diff
@@ -89,7 +89,7 @@ public T pop() {
    * @return true if the element removed successfully, false otherwise
    */
   public boolean remove(T element) {
-    return mQueue.remove(element);
+    return mQueue.removeIf(x -> x.mValue == element);
```

Comment:
NIT: use `getValue()` instead of accessing private member?

Reply:
Done.